dymos/models/atmosphere/atmos_1976.py

```diff
@@ -1341,11 +1357,12 @@ def compute_partials(self, inputs, partials):
         d2rho_dh2 = coeffs[:, 1] + dx * (2.0 * coeffs[:, 2] + 3.0 * coeffs[:, 3] * dx)
 
         partials['temp', 'h'] = dT_dh.ravel()
```

Comment:
partials['temp', 'h'] is in here twice.

Reply:
Once for the assignment, once in use by `partials['sos', 'h']` and once when using the chain rule if `_geodetic` is True.  Those are all necessary I think.